components/cam/src/dynamics/se/inidat.F90

```diff
@@ -453,18 +453,14 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
 
     if ( (ideal_phys .or. aqua_planet)) then
        tmp(:,1,:) = 0._r8
+       phis_tmp(:,:) = 0._r8
     else    
       fieldname = 'PHIS'
       tmp(:,1,:) = 0.0_r8
       if (fv_nphys > 0) then
-         ! Copy phis field to GLL grid
+         ! Load phis field to GLL grid first
```

Comment:
I believe this comment is incorrect when separated from the fv_phys_to_dyn_topo function calls. Here, infld is loading physgrid topo data, not GLL data.

Reply:
Right, I'll correct that.